pkg/migrations/sql/idempotent/base.sql

```diff
@@ -2497,7 +2582,15 @@ BEGIN
             remaining_metrics := remaining_metrics || r;
             CONTINUE;
         END IF;
+        IF log_verbose THEN
+            startT := clock_timestamp();
+            RAISE LOG 'promscale maintenance: compression: metric %: starting, without lock wait', r.metric_name;
+        END IF;
```

Comment:
I see this a general trend across some functions. Why not make a function that marks the start of time like `markTaskStart(message TEXT) RETURNS TIMESTAMPTZ`, and when the job is done, we send that mark `markTaskComplete(message TEXT, begin TIMESTAMPTZ)` and pass the returned timestamp into this function.

This will reduce redundancy here.

Reply:
Function call is PG are more expensive then I'd like to use for logging and I don't think it will make the code much cleaner